File: Java/EvenOdd.java

//Write a program to check whether entered number is even or odd
import java.io.*;
class EvenOdd
{
	public static void main(String args[])
	{
		int a;
		a=Integer.parseInt(args[0]);
		if(a%2==0)
			System.out.println("The No. is even");
		else
			System.out.println("The No. is odd");
	}
}

/*
Output:
23
Number is odd
54
Number is even
*/

File: Rust/evenodd.rs
use std::io;
use std::io::Write;

fn main() {
	let mut input = String::new();
	print!("Enter a number: ");	
	io::stdout().flush().unwrap();	
	match io::stdin().read_line(&mut input) {
		Ok(_) => {
			let number: i32 = input.trim().parse().unwrap();
			if number % 2 == 0 {
				println!("Even number");
			}
			else {
				println!("Odd number");
			}
		},
		Err(e) => println!("Error: {}",e)
	}
}
